frequently-encountered-problems.html

```diff
@@ -200,6 +200,39 @@ <h2>Using NTLM authentication with Node.js 17</h2>
         </p>
     </dd>
     <br />
+    <dt>
+        <h2>The relations between <code>options.trustServerCertificate, options.encrypt,</code> and <code>options.cryptoCredentialDetails</code></h2>
+    </dt>
+    <dd>
+        <ul>
+            <li>
+                If <code>encrypt = true</code> and <code>trustServerCertificate = false</code>, then <code>cryptoCredentialsDetails</code> must be provided. 
+                Details on how to provide the <code>cryptoCredentialsDetails</code> object can be found in the <a href="/api-connection.html">connection</a> page of the documentation.
+            </li>
+            <li>
+                If <code>encrypt = false</code>, then <code>trustServerCertificate</code> is ignored.
```

Comment:
This is what I found when we has the discussion with David, seems the current logic will attempt to check the certificate only based on the server-side settings. Not 100% sure, but you can double-check the logic, see if I got anything wrong. 

If server-side has set to encrypt to 'ON' or 'REQ', encrypt is set to true.  The driver will start TLS  with corresponding trustservercertificate settings.
If server-side has set to encrypt to 'ON' or 'REQ', and encrypt is set to false. The driver will throw an error with the message "Server requires encryption, set 'encrypt' config option to true." and close the connection.

Reply:
Added to the FAQ